backend/src/main/java/ai/verta/modeldb/advancedService/AdvancedServiceImpl.java

```diff
@@ -204,7 +204,7 @@ public AdvancedServiceImpl(
               selfAllowedActions.get(project.getId()).getActionsList());
         }
       } else {
-        LOGGER.error(ModelDBMessages.USER_NOT_FOUND_ERROR_MSG, project.getOwner());
+        LOGGER.warn(ModelDBMessages.USER_NOT_FOUND_ERROR_MSG, project.getOwner());
```

Comment:
since this is related to owner printing it as warn, but a user not found is not an error for MDB